backend/Watchdog.Core/Watchdog.Core.BLL/Services/OrganizationService.cs

```diff
@@ -29,14 +30,26 @@ public async Task<ICollection<OrganizationDto>> GetAllOrganizationsAsync()
             return _mapper.Map<ICollection<OrganizationDto>>(organizations);
         }
 
-        public async Task<OrganizationDto> CreateOrganizationAsync(OrganizationDto organizationDto)
+        public async Task<OrganizationDto> CreateOrganizationAsync(NewOrganizationDto organizationDto)
         {
             var organization = _mapper.Map<Organization>(organizationDto);
 
-            var createdUser = _context.Organizations.Add(organization);
+            await _context.Organizations.AddAsync(organization);
+
+            var member = new Member
+            {
+                UserId = organizationDto.CreatedBy,
+                CreatedBy = organizationDto.CreatedBy,
+                CreatedAt = DateTime.Now,
+                Organization = organization,
+                RoleId = 1,
```

Comment:
It would be better to find role which you want and after that set it Id

Reply:
Done